### PR TITLE
Fast Zaps ⚡ 

### PR DESCRIPTION
**Pull request by v0l**
Allows instant zapping with default zap amount

Long press zap icon to get old LNURL flow

### CODE REVIEW

**Review by verbiricha — APPROVED** [review]
⚡

File: packages/app/src/Element/NoteFooter.tsx [hunk 1]
```diff
@@ -102,15 +118,39 @@ export default function NoteFooter(props: NoteFooterProps) {
...
+    if (zapping || e.isPropagationStopped()) return;
+
+    const lnurl = author?.lud16 || author?.lud06;
+    if (webln?.enabled && lnurl) {
+      setZapping(true);
+      try {
+        const handler = new LNURL(lnurl);
+        await handler.load();
+
+        const zap = await publisher.zap(prefs.defaultZapAmount * 1000, ev.PubKey, ev.Id);
```
Comment (v0l) [inline packages/app/src/Element/NoteFooter.tsx:131]: Need to check `handler.canZap` should we allow Non-zap with fast button?

File: packages/app/src/LNURL.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        throw new Error("Invalid LNURL: not a url");
+      }
+      this.#url = new URL(decoded);
+    } else if (lnurl.match(EmailRegex)) {
+      const [handle, domain] = lnurl.toLowerCase().split("@");
+      this.#url = new URL(`https://${domain}/.well-known/lnurlp/${handle}`);
+    } else if (lnurl.toLowerCase().startsWith("http")) {
+      this.#url = new URL(lnurl);
+    } else {
+      throw new Error("Invalid LNURL: could not determine service url");
```
Comment (v0l) [inline packages/app/src/LNURL.ts:24]: Need to extract these messages

File: packages/app/src/Element/NoteFooter.tsx (hunk 1 above)
Comment (verbiricha) [reply packages/app/src/Element/NoteFooter.tsx:131]: I think falling back to non-zap if not supported is fine, still want the instant UX imo.

**Pull request merged**